app-i18n/rime-data/rime-data-1.0.20241201-r1.ebuild

```diff
@@ -72,7 +72,9 @@ src_install() {
 		for pkg in "${!LIST[@]}"; do
 			doins "${pkg}-${LIST[$pkg]}"/*[!AUTHORS\|LICENSE\|README.md\|check.py]
 			for f in AUTHORS LICENSE README.md; do
-				newdoc "${pkg}-${LIST[$pkg]}/${f}" "${pkg}_${f}"
+				if [ -f "${pkg}-${LIST[$pkg]}/${f}" ]; then
```

Comment:
Please use bash tests in general (`[[` and `]]`). But will merge now.

Reply:
Thanks. I'll modify it for the next bump then.